File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/LoginUserRequest.java
package com.rolandvitezhu.todocloud.network.api.user.dto;

import com.google.gson.annotations.SerializedName;

public class LoginUserRequest {

  @SerializedName("email")
  public String email;
  @SerializedName("password")
  public String password;

  public String getEmail() {
    return email;
  }

  public void setEmail(String email) {
    this.email = email;
  }

  public String getPassword() {
    return password;
  }

  public void setPassword(String password) {
    this.password = password;
  }
}

File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/LoginUserRequest.kt
package com.rolandvitezhu.todocloud.network.api.user.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class LoginUserRequest (
        @SerializedName("email")
        var email: String?,
        @SerializedName("password")
        var password: String?
) : Parcelable {
    constructor() : this(
            null,
            null
    )
}